### DISCUSSION TITLE
Turning all WARNs into FAILs

### DISCUSSION THREAD

**Discussion by khaledhosny** [fonttools/fontbakery#4099] [Q&A] [answered]
For a project I don’t want any WARNs to creep in, so I’d rather turn all of them into FAILs (I use a makefile, so I want `fontbakery` command to fail if there is a WARN, I don’t care if the report would show it as a WARN or as a FAIL).

Is there a command line option or a configuration setting to do this?

**Comment by felipesanches** [comment, marked as answer]
We currently don't have that. But it sounds like a useful thing to have, so we should consider implementing such a flag.

**Reply by khaledhosny** [reply]
With latest fontbakery, `-e WARN` does the trick.
